stories/10-For Developers/Token Implementation Notes.stories.mdx

```diff
@@ -0,0 +1,229 @@
+
+There's also a set of default line-height tokens to match each font size token. Because ideal default line heights for fonts can vary depending on the typeface and context, these need to be configured manually.
+
+When a developer uses a `--font-size-*` token they should generally use the correspending `--line-height-*` token, unless manually setting a different height makes sense (for example, single-line text that should be vertically centered, such as in a button or tag, might have a custom line height to match the container size.)
+
+### The typeface mixin
+To help developers always include the font-weight, letter-spacing, and line-height tokens when setting a font, the typeface mixin sets all these values in a single line.
+
+```
+@include typeface(header, 10)
+```
+will expand to
+```
+font-family:    var(--font-family-header);
+letter-spacing: var(--letter-spacing-header);
+font-weight:    var(--font-weight-header);
+font-size:      var(--font-size-10);
+line-height:    var(--line-height-10);
+```
+
+A recommended pattern is to use the typeface mixin whenever setting a typeface for a component, element or section, but then use only the font size and line height tokens in media queries or modifier classes that adjust the size.
+
+Example font token usage:
+```
+.my-header {
+  // the base font size. This is "mobile-first".
+  @include typeface (header, 8);
+
+    @include media(">=medium") {
+      // increase the font size on a wider screen
+      font-size:      var(--font-size-10);
+      line-height:    var(--line-height-10);
+    }
+}
+
+.my-header.mod-smaller {
+    font-size:      var(--font-size-6);
+    line-height:    var(--line-height-6);
+
+    @include media(">=medium") {
+       font-size:      var(--font-size-8);
+       line-height:    var(--line-height-8);
+    }
+}
+```
+
+```
+// DON'T DO THIS
+.my-button {
+  @include typeface (body, 6);
+  line-height: 100%; // customized line height for buttons.
+}
+
+.my-button.mod-small {
+  @include typeface (body, 4); // overwrites the customized line height. oops!
+}
+```
+
+## Spacing
+
+Radial includes three predefined spacing scales, Compact, Default, and Loose. To choose the spacing scale for a theme, the theme author can change the variables `--spacing-base` and `spacing-increment` to use a pair of these predefined variables.
+
+Setting a theme to use compact spacing:
+```
+  --spacing-base:      var(--spacing-base-compact);
+  --spacing-increment: var(--spacing-increment-compact);
+```
+
+Setting these levers configures the values for a range of spacing tokens, numbered `--space-1` to `--space-11`.
+
+When creating components, developers and designers should use these tokens consistely according to the spacing guidelines.
+
+## Colors
+
+### Color Format
+
+One nice feature of SASS is its built in color functions. You can write `rgba(#ff000, 0.8)` and it will become `rgba(255, 0, 0, 0.8)`. Or more usefully you might write something like `rgba($theme-red, $hover-opacity)`.
```

Comment:
```suggestion
One nice feature of SASS is its built-in color functions. You can write `rgba(#ff000, 0.8)` and it will become `rgba(255, 0, 0, 0.8)`. Or more usefully you might write something like `rgba($theme-red, $hover-opacity)`.
```